packages/forms-web-app/src/controllers/selected-appeal/selected-appeal.js

```diff
@@ -54,15 +54,40 @@ exports.get = async (req, res) => {
 
 	const viewContext = {
 		appeal: {
-			appealNumber: appealNumber,
+			appealNumber,
 			headlineData,
-			// placeholder sections info
 			sections: userSections[userType].map((section) => ({
 				...section,
 				links: section.links.filter(({ condition }) => condition(caseData))
-			}))
+			})),
+			decision: formatDecision({ ...caseData, caseDecisionOutcome: 'invalid' })
```

Comment:
Won't this override the `caseData.caseDecisionOutcome` value?

Reply:
oops yes didn't mean to commit that, was just gathering screenshots to show Steve